src/popup/components/Modals/MultisigVaultCreate.vue

```diff
@@ -26,8 +26,8 @@
           v-model.trim="signer.address"
           :default-text="$t('modals.createMultisigAccount.signerInputPlaceholder')"
           :label="getSignerLabel(index)"
-          item-title="address"
           :options="aeAccountsSelectOptions"
+          item-title="value"
```

Comment:
what is `value` in this context?

Reply:
This is a common concept for custom Vue select components where the items in the list are objects. So you are able to define which object property (key) is used as the title and the value. Reference: https://vuetifyjs.com/en/api/v-select/#props-item-title